omod/src/main/webapp/resources/htmlforms/HMIS-ACP-005b-Emergency-ART-Refill-Form.xml

```diff
@@ -0,0 +1,228 @@
+<htmlform formName="HMIS ACP 005b: Emergency ART Refill Form" 
+formDescription="HMIS ACP 005b: Emergency ART Refill Form" 
+formEncounterType="d18bd8f2-dfcd-11eb-ba80-0242ac130004" 
+formUuid="aa858f72-0080-44a8-8bac-55bc275e1d10" 
+formVersion="3.0" 
+formAddMetadata="yes" 
+formUILocation="patientDashboard.visitActions" 
+formOrder="3" 
+formIcon="icon-medkit" 
+formShowIf="(visit.active || !visit.active) &amp;&amp; patient.person.dead==false &amp;&amp; (sessionLocation.uuid=='86863db4-6101-4ecf-9a86-5e716d6504e4' || sessionLocation.uuid=='7c231e1a-1db5-11ea-978f-2e728ce88125')"
+formDisplayStyle="Standard">
+
+
+    <script type="text/javascript">
+
+            if (jQuery) {
+    
+                jq(document).ready(function () {   
+                    
+                //Block Multiple encounter entry on same date
+                blockEncounterOnSameDateEncounter(jq("#encounterDate").find("input[type=hidden]"),"block");                        
+                    
+                beforeSubmit.push(function () {
+                    var submitVal = true;
+
+                if(getValue('reason-for-visit.value')==""){
+                    jq().toastmessage('showErrorToast', "Reason for visit has not been added");
+                }
+    
+                if(getValue('current-regimen.value')==""){
+                    jq().toastmessage('showErrorToast', "Current Regimen has not been added");
+                }
+
+                if(getValue('parent-health-facility.value')==""){
+                    jq().toastmessage('showErrorToast', "Add the Parent Health Facility as required");
+                }
+
+                if(getValue('serial-number.value')==""){
+                    jq().toastmessage('showErrorToast', "Add the Serial Number as required");
+                }
+
+                if(getValue('pt-clinic-number.value')==""){
```

Comment:
@ODORA0 This should be optional if the age is below 5 years when the EID number is shown but required for above 5 years. Use <IncludeIF for age

Reply:
@ssmusoke Done

Reply:
@ssmusoke Done